cmd/relay-pls/main.go

```diff
@@ -26,7 +34,21 @@ func main() {
 		}
 	}
 
-	var serverOpts []server.BigQueryServerOption
+	if cfg.MetricsEnabled {
```

Comment:
It looks like `WithMetrics` checks `MetricsEnabled` itself, so is this check superfluous?

Reply:
Yep.